StellarAssetSdk/AssetIssuer.cs

```diff
@@ -175,7 +176,23 @@ public async Task<Transaction> CreateClaimableBalanceTransaction(Account sender,
      */
     public async Task<Transaction> ClaimClaimableBalanceTransaction(Account receiver, byte[] claimableBalanceId)
     {
-        // TODO: use example above
-        throw new NotImplementedException("ClaimClaimableBalanceTransaction");
+        // Adding some zeros to match the real claimable balance id
+        var claimableIdHex = "00000000" + claimableBalanceId.ToStringHex();
```

Comment:
```suggestion
        var claimableIdHex = claimableBalanceId.ToStringHex();
```
Tested this and this isn't necessary.